nntrainer/layers/lstmcell.cpp

```diff
@@ -32,13 +31,7 @@ enum LSTMCellParams {
 
 LSTMCellLayer::LSTMCellLayer() :
   LayerImpl(),
-  lstmcell_props(props::Unit(), props::IntegrateBias(),
-                 props::HiddenStateActivation() = ActivationType::ACT_TANH,
-                 props::RecurrentActivation() = ActivationType::ACT_SIGMOID,
-                 props::DropOutRate()),
-  acti_func(ActivationType::ACT_NONE, true),
-  recurrent_acti_func(ActivationType::ACT_NONE, true),
-  epsilon(1e-3) {
+  lstmcell_props(props::DropOutRate()) {
```

Comment:
so, only dropout rate is the property for LSTMCell? and others ( Activation for Hidden and Recurrent & epsilon ) are for the LSTMCore? Could you add some comment about it?

Reply:
Added